Shell/Options.cpp

```diff
@@ -133,6 +133,10 @@ void Options::init()
     _lookup.insert(&_mode);
     _mode.addHardConstraint(If(equal(Mode::CONSEQUENCE_ELIMINATION)).then(_splitting.is(notEqual(true))));
 
+    auto UsingCascTechnology = [this] {
```

Comment:
Is this CASC technology or Portfolio technology? I think the latter.  Perhaps add a comment next to the mode enum saying that this may need updating if a new portfolio-based mode is added.